apps/Gruntfile.js

```diff
@@ -342,6 +342,10 @@ describe('entry tests', () => {
             'build/package/css/code-studio.css',
             'style/code-studio/code-studio.scss'
           ],
+          [
+            'build/package/css/certificates.css',
+            'style/curriculum/certificates.scss'
```

Comment:
I see mentions of these css files, but I don't see the file itself. Are you planning to add it later on? I'm only seeing inline styles

Reply:
Ah, I see it now! Ignore me